packages/api/pages/api/verse.ts

```diff
@@ -0,0 +1,39 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import { client } from '../../db'
+
+export default async function(req: NextApiRequest, res: NextApiResponse) {
+  const verse = await client.verse.findFirst({
+    where: {
+      number: 1,
+      chapter: {
+        number: 1,
+        book: {
+          search: 'matthew'
+        },
+      } 
+    },
+    select: {
+      number: true,
+      chapter: {
+        select: {
+          number: true,
+          book: {
+            select: {
+              name: true
+            }
+          }
+        }
+      }
+    }
+  })
```

Comment:
And here is it in use. We can select a verse based on its foreign keys in multiple tables, and prisma can decide how it wants to build the query or queries it needs. Prisma exposes higher level query functions rather than trying to map exactly to SQL. This query here actually results in three queries for the verse, chapter, and book. We can always drop into a SQL command if we want to though.